packages/dev/core/src/scene.ts

```diff
@@ -1012,8 +1017,39 @@ export class Scene extends AbstractScene implements IAnimatable, IClipPlanesHold
         return this._lightsEnabled;
     }
 
+    private _activeCameras: Nullable<Camera[]> = new Array<Camera>();
     /** All of the active cameras added to this scene. */
-    public activeCameras: Nullable<Camera[]> = new Array<Camera>();
+    public get activeCameras(): Nullable<Camera[]> {
+        return this._activeCameras;
+    }
+
+    private _proxySet(target: Camera[], property: string | symbol, c: Camera) {

```

Comment:
_proxySet/push/makeObs... would be better in their own class or with a specific name for cameras, also I wonder if it could not be fully shared with the renderTarget implementations of _hookArray ?

Reply:
the proxy will be abstracted and moved to tools to share the implementation